airflow/models/dag.py

```diff
@@ -1781,7 +1781,8 @@ def create_dagrun(
 
         :meta private:
         """
-        logical_date = timezone.coerce_datetime(logical_date)
+        if logical_date is not None:
+            logical_date = timezone.coerce_datetime(logical_date)
```

Comment:
```suggestion
        logical_date = timezone.coerce_datetime(logical_date)
```

Does not need to change this; `coerce_datetime` can handle None correctly.

Reply:
Implemented